### ISSUE TITLE
CustomFAB problems

### ISSUE THREAD

**Issue by MaxiStefan** [theiskaa/flutter_button#4]
is there any way to work around using more than 1 CustomFAB? i get the multiple heroes in a subtree error and i do not see any heroTag under the widget's list of properties. Thanks!

**Comment by theiskaa** [comment]
First of all, thanks so much for the first issue of the `flutter_button` package. Yeah, i guess you are right, the problem is we didn't provide any heroTah under the `CustomFAB`,  so to fix this issue you can contribute `flutter_button` and fix this problem. or you have to wait for `flutter_button`'s new version. 
Thanks again!

**Comment by Steve55jobs** [comment]
@theiskaa i saw the develop branch, and here `CustomFAB` was removed right? what you planning? what is next `CustomFAB`?

**Comment by theiskaa** [comment]
@Steve55jobs yeah Steve in the next version of the `flutter_button` package, I would remove more buttons that are not relevant. so they are simple, everyone can make it without a package. so soon you can see the cleaned `flutter_button` package
